tests/Aspire.Hosting.Azure.Tests/AzureBicepResourceTests.cs

```diff
@@ -732,7 +732,7 @@ public async Task WithReferenceAppInsightsSetsEnvironmentVariable()
 
         appInsights.Resource.Outputs["appInsightsConnectionString"] = "myinstrumentationkey";
 
-        var serviceA = builder.AddProject<ProjectA>("serviceA")
+        var serviceA = builder.AddProject<ProjectA>("serviceA", o => o.ExcludeLaunchProfile = true)
```

Comment:
Just curious - why is this needed now?

Reply:
See https://github.com/dotnet/aspire/issues/4877